src/field/broken-project/index.md

```diff
@@ -0,0 +1,26 @@
+# How to Fix a Broken Project
+<Badge text="since Input 2.0.0" type="tip"/>
+[[toc]]
+
+Do you get an error message when trying to open a project in <MobileAppName />? There can be multiple reasons for what went wrong. Here we will try to guide you through some basic steps that can help you.
+
+![error dialog](./error-dialog.gif)
+
+Since <MobileAppName /> version 2.0.0, you may encounter an error when working with QGIS projects that were created in older versions in QGIS. It is caused by a bug that should be fixed in QGIS versions 3.20, 3.18.3 and 3.16.7 and higher.
```

Comment:
```suggestion
Since <MobileAppName /> version 2.0.0, you may encounter an error when working with QGIS projects that were created in older versions of QGIS. It is caused by a bug that should be fixed in QGIS versions 3.20, 3.18.3 and 3.16.7 and higher.
```